firefox-settings/pure.js

```diff
@@ -155,3 +155,37 @@ pref("browser.safebrowsing.provider.mozilla.pver", 0, locked);
 pref("browser.safebrowsing.provider.mozilla.reportURL", "", locked);
 pref("browser.safebrowsing.provider.mozilla.updateURL", "", locked);
 pref("browser.safebrowsing.reportPhishURL", "", locked);
+
+pref("browser.contentblocking.category", "strict"); // https://blog.mozilla.org/security/2021/02/23/total-cookie-protection/
+pref("browser.download.autohideButton", false, locked); //do not hide download button automatically
+pref("browser.download.start_downloads_in_tmp_dir", true, locked); //start downloads in temp
+pref("browser.helperApps.deleteTempFileOnExit", true); // https://bugzilla.mozilla.org/show_bug.cgi?id=238789#c19
+pref("browser.tabs.unloadOnLowMemory", true, locked); // Unload unused tabs
+pref("browser.uitour.enabled", false); // https://trac.torproject.org/projects/tor/ticket/19047#comment:3
```

Comment:
I prefer it to be locked.
```suggestion
pref("browser.uitour.enabled", false, locked); // https://trac.torproject.org/projects/tor/ticket/19047#comment:3
```